dcpy/lifecycle/package/abstract_doc.py

```diff
@@ -0,0 +1,360 @@
+from pathlib import Path
+from typing import Any
+
+from dcpy.models.design import elements as de
+from dcpy.models.product.dataset.metadata_v2 import Dataset
+from dcpy.models.product.metadata import OrgMetadata
+from dcpy.models.product.artifacts import Artifact, ComponentDefinition
+from dcpy.utils.logging import logger
+
+
+# TODO; Extract these into a generic style that we can pass to the XLSX renderer
+BLUE = "FF009DDC"
+TITLE_FONT_SIZE = 18.0
+MONOSPACED_FONT = "Consolas"
+
+
+def _make_title_subtitle_cell(title: str, subtitle: str):
```

Comment:
@fvankrieken some additional context on a previous comment: this type of logic is what I'm thinking we'll move into the XLSX module, along with basically everything in _make_table_top. I think it's really only useful when made concrete in a renderer.